src/main/java/ru/sbtqa/tag/pagefactory/exceptions/ElementValueException.java

```diff
@@ -0,0 +1,30 @@
+package ru.sbtqa.tag.pagefactory.exceptions;
```

Comment:
This exception also looks excess for me. It is being thrown only in one place (WebExtension:32):
```
	if (elementId == null) {
	    throw new ElementValueException("Getting value is not support in element without id");
	}
```
`IllegalArgumentException` fits well enough here, why not use it?